Chapters/01-zig-weird.qmd

```diff
@@ -130,70 +109,38 @@ tree .
 1 directory, 4 files
 ```
 
+The `ìnit` command creates two additional files in the work directory - `build.zig` and `build.zig.zon`.
+`build.zig` represents the build script for your project and it is written in Zig (consistency!).
+It will be executed with the command `zig build`.
+You won't need an external build system when using Zig. The file `build.zig` brings all the code necessary to build the entire project.
 
-Now, in addition to the source directory, two other files were created in our working directory:
-`build.zig` and `build.zig.zon`. The first file (`build.zig`) represents a build script written in Zig.
-This script is executed when you call the `build` command from the `zig` compiler.
-In other words, this file contain Zig code that executes the necessary steps to build the entire project.
-
-In general, low-level languages normally use a compiler to build your
-source code into binary executables or binary libraries.
-Nevertheless, this process of compiling your source code and building
-binary executables or binary libraries from it, became a real challenge
-in the programming world, once the projects became bigger and bigger.
-As a result, programmers created "build systems", which are a second set of tools designed to make this process
-of compiling and building complex projects, easier.
-
-Examples of build systems are CMake, GNU Make, GNU Autoconf and Ninja,
-which are used to build complex C and C++ projects.
-With these systems, you can write scripts, which are called "build scripts".
-They simply are scripts that describes the necessary steps to compile/build
-your project.
-
-However, these are separate tools, that do not
-belong to C/C++ compilers, like `gcc` or `clang`.
-As a result, in C/C++ projects, you have not only to install and
-manage your C/C++ compilers, but you also have to install and manage
-these build systems separately.
-
-But instead of using a separate build system, in Zig, we use the
-Zig language itself to write build scripts.
-In other words, Zig contains a native build system in it. And
-we can use this build system to write small scripts in Zig,
-which describes the necessary steps to build/compile our Zig project[^zig-build-system].
-So, everything you need to build a complex Zig project is the
-`zig` compiler, and nothing more.
+<!-- why explaining details of other langugages? This book is about Zig, not a general introduction into programming -->

```

Comment:
Once again, making comparisons is important for those that come from other languages, and are learning Zig as the second, third, or fourth language, etc.